modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -497,7 +497,10 @@ public override void _EnablePlugin()
 
             AddChild(new HotReloadAssemblyWatcher { Name = "HotReloadAssemblyWatcher" });
 
-            _menuPopup = new PopupMenu();
+            _menuPopup = new PopupMenu
+            {
+                Name = "CSharpTools",
+            };
```

Comment:
I don't know what's the preferred style but I notice the `AddChild` function above does this inline:
```suggestion
            _menuPopup = new PopupMenu { Name = "CSharpTools" };
```

Reply:
Maybe it's supposed to be inlined when it's just one property? cc @raulsntos

Reply:
Yes, I think we tend to inline them when it's just one property and the line is not overly long. But we don't really enforce this so we're probably not consistent everywhere. I think it's fine either way.